src/vorta/borg/create.py

```diff
@@ -51,13 +51,13 @@ def finished_event(self, result):
 
     @classmethod
     def pre_post_backup_cmd(cls, params, cmd='pre_backup_cmd', returncode=0):
-        cmd = getattr(params['profile'], cmd)
+        cmd = getattr(params['profile_export'], cmd)
```

Comment:
Search & Replace gone overboard here? Also affects a few other files.

Reply:
Oh how embarassing. Fixing it ASAP.

Reply:
No worries. Can happen in the heat of battle.